src/fileActions.jsx

```diff
@@ -228,6 +228,17 @@ export const CreateDirectoryModal = ({ selected, currentPath }) => {
         spawnCreateDirectory({ Dialogs, selected, currentPath, name, setErrorMessage });
     };
 
+    useEffect(() => {
+        const onEnter = e => {
+            if (e.key === "Enter") {
+                e.preventDefault();
+                createDirectory();
+            }
+        };
+        document.addEventListener("keydown", onEnter);
+        return () => document.removeEventListener("keydown", onEnter);
+    });
+
     return (
         <Modal
           position="top"
```

Comment:
The way you did it has one issue I can think of. Even if you focus on the Close or Cancel button and hit enter the form will be submitted looks like.

The usual way to do that is do add an onSubmit callback to the Form.

Something like:
```
onSubmit={event => { createDirectory(); event.preventDefault(); return false }}>
```

Reply:
Changed, thanks!